src/cmd.rs

```diff
@@ -55,11 +55,105 @@ pub struct MgrMetadata {
     pub hostname: String,
     pub kernel_description: String,
     pub kernel_version: String,
-    pub mem_swap_kb: u64,
-    pub mem_total_kb: u64,
+    pub mem_swap_kb: String,
```

Comment:
I seem to recall ceph changing this field type somewhere along the way.  What version did you test against for this?

Reply:
Luminous

Reply:
Actually, I could probably make this an enum, so yanno, either it gets serialized as u64 or String.  hm.

Reply:
I should probably also make those struct attributes public too

Reply:
Eh actually wait that's implied